module/modifier-bucket/bucket-app.js

```diff
@@ -841,11 +830,17 @@ export class ModifierBucket extends Application {
     }
     return content
   }
+  /**
 
-  _injectHTML(html) {
-    if ($('body').find('#bucket-app').length === 0) {
-      html.insertAfter($('body').find('#hotbar'))
-      this._element = html
-    } else console.warn('=== HOLA ===\n That weird Modifier Bucket problem just happened! \n============')
+     * @override
+     */
+  _injectHTML($html) {
```

Comment:
There's now a new system setting to move the modifier bucket to the right (next to chat bar). Make sure it still works.